src/input/platform_input.cpp

```diff
@@ -0,0 +1,8 @@
+#include "platform_input.h"
+
+platf::input_t PlatformInput::_instance;
+
+platf::input_t &
+PlatformInput::getInstance() {
+  return _instance;
+}
```

Comment:
```suggestion
}

```

I probably need to a workflow to check for this.

Reply:
Idk why CLion failed to format the end-line... It did to other files.